pkgs/top-level/python-packages.nix

```diff
@@ -21572,7 +21572,7 @@ in modules // {
 
   basemap = buildPythonPackage rec {
     name = "basemap-1.0.7";
-    disabled = ! isPy27;
+    disabled = ! (isPy27 || isPy3k);
 
```

Comment:
So for which interpreter version is it supposed to work then? Since there aren't many left this way ;-)
I guess you just want to remove the `disabled = ! isPy27;` line altogether.